library/gui/screen/src/main/java/org/quiltmc/qsl/screen/impl/client/ScreenExtensions.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.screen.impl.client;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.quiltmc.qsl.base.api.event.Event;
+import org.quiltmc.qsl.screen.api.client.ScreenEvents;
+import org.quiltmc.qsl.screen.api.client.ScreenKeyboardEvents;
+import org.quiltmc.qsl.screen.api.client.ScreenMouseEvents;
+
+import net.minecraft.client.gui.screen.Screen;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@ApiStatus.Internal
+@Environment(EnvType.CLIENT)
+public interface ScreenExtensions {
```

Comment:
`InternalScreenExtensions`?

Reply:
👍

Reply:
Btw. This interface will be removed in the refactor im working on based on pepper's review, so... it doesn't matter what this interface is called